Example/UI Examples/BrowseViewController.swift

```diff
@@ -0,0 +1,179 @@
+//
+//  BrowseViewController.swift
+//  UI Examples
+//
+//  Created by Ben Guo on 7/18/17.
+//  Copyright © 2017 Stripe. All rights reserved.
+//
+
+import UIKit
+import Stripe
+
+class BrowseViewController: UITableViewController, STPAddCardViewControllerDelegate, STPPaymentMethodsViewControllerDelegate, STPShippingAddressViewControllerDelegate {
+
+    enum Demo: String {
+        static let count = 5
+        case STPPaymentCardTextField = "Card Field"
+        case STPAddCardViewController = "Card Form"
+        case STPPaymentMethodsViewController = "Payment Methods"
+        case STPShippingInfoViewController = "Shipping Info"
+        case ChangeTheme = "Change Theme"
+
+        init?(row: Int) {
+            switch row {
+            case 0: self = .STPPaymentCardTextField
+            case 1: self = .STPAddCardViewController
+            case 2: self = .STPPaymentMethodsViewController
+            case 3: self = .STPShippingInfoViewController
+            case 4: self = .ChangeTheme
+            default: return nil
+            }
+        }
+    }
+
+    let customerContext = MockCustomerContext()
+    let themeViewController = ThemeViewController()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.title = "UI Examples"
+        self.tableView.tableFooterView = UIView()
+        self.navigationController?.navigationBar.isTranslucent = false
```

Comment:
My style has usually been explicit `self`, since it visually distinguishes from local vars (this is what we do in the dashboard), but I'm starting to come around – syntax highlighting provides enough clarity. I'll update.